cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -121,11 +121,19 @@ void gpi_log(const char *name, long level, const char *pathname, const char *fun
             if (0 > n) {
                fprintf(stderr, "Log message construction failed\n");
             }
- 
+
             fprintf(stdout, "     -.--ns ");
             fprintf(stdout, "%-9s", log_level(level));
             fprintf(stdout, "%-35s", name);
-            fprintf(stdout, "%20s:", pathname);
+
+            n = strlen(pathname);
+            if (20 < n) {
```

Comment:
nit: `n > 20` is easier to read. (I know this file does it differently in other places, but its still good to fix that here)

Reply:
I also fixed the `0 > n` in the same function.